lib/Model/Listing/AbstractListing.php

```diff
@@ -230,7 +231,7 @@ public function setOrderKey($orderKey, $quote = true)
 
         if (is_array($orderKey)) {
             foreach ($orderKey as $o) {
-                if ($quote === false) {
+                if (!$backticks || $this->checkIfValueContainBackticks($o)) {
                     $this->orderKey[] = $o;
                 } elseif ($this->isValidOrderKey($o)) {
                     $this->orderKey[] = '`' . $o . '`';
```

Comment:
better use `$db->quote` here?

Reply:
make sense, i will change it

Reply:
@dpfaffenbauer do you recommend to use $db = Db::get(); or implement the ConnectionInterface? i'm not sure if the ConnectionInterface via autowire is working and $db = Db::get(); is the ugliest solution :)

Reply:
only option here is `DB:get`.

Reply:
done